Source/WebKit/Scripts/webkit/model.py

```diff
@@ -135,6 +135,6 @@ def generate_global_model(receivers):
         for message in receiver.messages:
             if message.reply_parameters is not None and not message.has_attribute(SYNCHRONOUS_ATTRIBUTE):
                 async_reply_messages.append(Message(name='%s_%sReply' % (receiver.name, message.name), parameters=message.reply_parameters, reply_parameters=[], attributes=None, condition=message.condition))
-    async_reply_receiver = MessageReceiver(name='AsyncReply', superclass='None', attributes=[BUILTIN_ATTRIBUTE], receiver_enabled_by=None, receiver_enabled_by_exception=False, receiver_enabled_by_conjunction=None, receiver_dispatched_from=None, receiver_dispatched_from_exception=None, receiver_dispatched_to=None, receiver_dispatched_to_exception=None, shared_preferences_needs_connection=False, messages=async_reply_messages, condition=None, namespace='WebKit', wants_send_cancel_reply=False)
+    async_reply_receiver = MessageReceiver(name='AsyncReply', superclass='None', attributes=[BUILTIN_ATTRIBUTE], receiver_enabled_by=None, receiver_enabled_by_exception=False, receiver_enabled_by_conjunction=None, receiver_dispatched_from=None, receiver_dispatched_from_exception=True, receiver_dispatched_to=None, receiver_dispatched_to_exception=True, shared_preferences_needs_connection=False, messages=async_reply_messages, condition=None, namespace='WebKit', wants_send_cancel_reply=False)
```

Comment:
Why are you changing the default of receiver_dispatched_from_exception from None to True?  That seems like a step in the wrong direction.

Reply:
Filed: https://bugs.webkit.org/show_bug.cgi?id=297326